LEGO1/mxstring.h

```diff
@@ -14,7 +14,9 @@ class MxString : public MxCore
   MxString(const char *);
   void ToUpperCase();
   void ToLowerCase();
-  const MxString &operator=(MxString *);
+  const MxString&  operator=(MxString *);
+  const MxString   operator+(const char *);
+  const MxString& operator+=(const char *);
```

Comment:
The `const ` is not needed for any of the return types.
It needlessly blocks modification of its return type.

Reply:
Fair point. I was kinda blindly following the use of `const` in the DLL exported operator